rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2534,3 +2552,143 @@ rocalResetLoaders(RocalContext p_context)
     }
     return ROCAL_OK;
 }
+
+RocalTensor  ROCAL_API_CALL
+rocalAudioFileSourceSingleShard(
+        RocalContext p_context,
+        const char* source_path,
+        const char* source_file_list_path,
+        unsigned shard_id,
+        unsigned shard_count,
+        bool is_output,
+        bool shuffle,
+        bool loop,
+        bool downmix,
+        unsigned max_frames,
+        unsigned max_channels,
+        unsigned storage_type,
+        bool stick_to_shard,
```

Comment:
Currently stick_to_shard & shard_size is not in use - To be removed.
Will be added as a part of the last batch policy

Reply:
done